src/EnergyPlus/ChillerReformulatedEIR.cc

```diff
@@ -1457,19 +1452,15 @@ namespace ChillerReformulatedEIR {
                     ShowContinueError(
                         "EIR as a function of PLR curve output at various part-load ratios and condenser water temperatures shown below:");
                     ShowContinueError("PLR           =    0.00   0.10   0.20   0.30   0.40   0.50   0.60   0.70   0.80   0.90   1.00");
-                    std::string StringVar; // Used for EIRFPLR warning messages
-                    ObjexxFCL::gio::write(StringVar, "'Cond Temp(C) = '");
-                    for (int CurveValPtr = 1; CurveValPtr <= 11; ++CurveValPtr) {
-                        ObjexxFCL::gio::write(StringVar, "(F7.2,$)") << CondTempArray(CurveValPtr);
-                    }
-                    ObjexxFCL::gio::write(StringVar);
-                    ShowContinueError(StringVar);
-                    ObjexxFCL::gio::write(StringVar, "'Curve Output = '");
-                    for (int CurveValPtr = 1; CurveValPtr <= 11; ++CurveValPtr) {
-                        ObjexxFCL::gio::write(StringVar, "(F7.2,$)") << CurveValArray(CurveValPtr);
-                    }
-                    ObjexxFCL::gio::write(StringVar);
-                    ShowContinueError(StringVar);
+
+                    const auto cond_temp_error = format("'Cond Temp(C) = ' {:7.2F}", fmt::join(CondTempArray, " "));
+                    std::cout << cond_temp_error << '\n';
+                    ShowContinueError(cond_temp_error);
+
+                    const auto curve_temp_error = format("'Curve Temp(C) = ' {:7.2F}", fmt::join(CurveValArray, " "));
+                    std::cout << curve_temp_error << '\n';
+                    ShowContinueError(curve_temp_error);
```

Comment:
@lefticus The attached test files exercise the negative eir error message for both ChillerEIR and ChillerReformulatedEIR. 
[7971-GIOChillerError.zip](https://github.com/NREL/EnergyPlus/files/4599975/7971-GIOChillerError.zip)


With v9.3.0 release, nothing is written to stdout (error messages like that never should be anyway). That said, the prior error message didn't work before (only one or two values were written to the err file). 

The new code fixes it, but the writes to std::cout should be removed here and in the other chiller.
Also, in this one, remove the single quotes around `Cond Temp(C) = ` in line 1456
And line 1460 should be `"Curve Output ="`
Thanks.

Reply:
OK, I'll remove those again. The old code definitely had something that was trying to write to stdout.

Reply:
@mjwitte does it make sense for GIOChillerError to become a regular test in the regression suite?

Reply:
@lefticus No, we try to keep the regression suite running with minimal errors. We could build a unit test for this. We have some coverage of errors in the unit tests, but it's a rather small percentage throughout the code.

Reply:
OK. It looks like I need to investigate a new failure on Windows and see if it's related to these changes. Hopefully we can get this PR wrapped up soon.